shared/actions/fs/index.js

```diff
@@ -967,12 +966,14 @@ const waitForKbfsDaemon = (state, action) => {
 const startManualCR = (state, action) =>
   RPCTypes.SimpleFSSimpleFSClearConflictStateRpcPromise({
     path: Constants.pathToRPCPath(action.payload.tlfPath),
-  }).then(() =>
-    FsGen.createTlfCrStatusChanged({
-      status: 'in-manual-resolution',
-      tlfPath: action.payload.tlfPath,
-    })
-  ) // TODO: deal with errors
+  }).then(() => FsGen.createFavoritesLoad()) // TODO: deal with errors
+// songgao: perhaps if this errors we should just let it black bar? I guess
+// depends on if/how we expect this to fail.
```

Comment:
Only situation I can that maybe shouldn't black bar is if the user is running multiple KBFSes and thus has a nil CR DB. I think black bar is probably fine though.